src/Resources/OracleTable.php

```diff
@@ -85,15 +85,14 @@ protected function getQueryResults(TableSchema $schema, Builder $builder, $extra
                 $result = [];
                 /** @var Collection $group */
                 foreach ($nestedResults as $group) {
-                    $record = array_only($group->first(), array_keys($otherFields));
+                    $record = \Illuminate\Support\Arr::only($group->first(), array_keys($otherFields));
```

Comment:
@daniilly Why are you using the FQCN here?

Reply:
@anas-srikou Corrected